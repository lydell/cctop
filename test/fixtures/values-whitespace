command          value     foo        |

command     a      b      |
\                        a          |
\b   |
