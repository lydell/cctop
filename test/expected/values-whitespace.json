{
  "commands": [
    {
      "name": "command",
      "value": "value     foo        ",
      "lines": [[1, 18, 21]],
      "line": 1,
      "column": 1,
      "endLine": 1,
      "subCommands": []
    },
    {
      "name": "command",
      "value": "a      b      \n                       a          \nb   ",
      "lines": [[3, 13, 14], [4, 3, 34], [5, 2, 4]],
      "line": 3,
      "column": 1,
      "endLine": 5,
      "subCommands": []
    }
  ],
  "errors": []
}